flink-python/dev/lint-python.sh

```diff
@@ -22,6 +22,8 @@
 #
 # You can refer to the README.MD in ${flink-python} to learn how easy to run the script.
 #
+# The file is directly downloaded and used by the different connector implementations
```

Comment:
Same deep copy suggestion here. In mid/long term this script can be modularized and not having a single script.